armbian/build
# Fix overlay configuration options for Orange Pi Zero 2 and Zero 3

# Description

This change includes kernel patches to provide the following overlays when using `armbian-config` on the Orange Pi Zero 2 and Orange Pi Zero 3 boards.

- I2C
- IR
- USB OTG
- SPI
- UART

These boards share very similar SOCs (RH616 / RH618), and the existing configuration that was also shared with the bigtreetech board did not function correctly for the Orange Pi Zero 2/3 boards, so the Zero 2/3 board overlay configuration was split out into its own files, and the board configurations updated to reference the new overlays.

Note: The patches for these overlays have been added to the 6.1, 6.6 (current), and 6.7 (edge) patch directories but it appears the 6.1 code may not be used with the current configurations for these boards.

Jira reference number [AR-9999]

# How Has This Been Tested?

These updates have been tested with the following configurations

- [x] Zero 2 Current and Edge kernel with Debian userspace
- [ ] Zero 3 Current and Edge kernel with Debian userspace (current pending testing)

The validation process at a high level
1. Boot clean Armbian image and configure base configuration
2. Launch `armbian-config` and turn on all Hardware overlays options for the board
3. Reboot when prompted, and confirm that overlays apply correctly in early boot logs (serial connection)
4. Confirm hardware devices are available in the OS

# Checklist:
